force-app/main/default/classes/ElevateAuthorizedGift.cls

```diff
@@ -64,6 +64,8 @@ public with sharing class ElevateAuthorizedGift {
         this.gatewayTransactionId = authorizedGiftResponse.gatewayTransactionId();
     }
 
+    public ElevateAuthorizedGift() {}
```

Comment:
Is this empty constructor needed?  Couldn't find any usages.

Reply:
During the `remove()` call on ElevateBatch.js, we pass an ElevateAuthorizedGift to `removeFromElevateBatch()` on GE_GiftEntryController.cls.

The server was expecting a no-argument constructor on ElevateAuthorizedGift in order for the deserialization to occur. Without this constructor, the method call was failing. I may be missing something here so please let me know if you have any recommendations.